feature/src/main/java/com/teamdontbe/feature/notification/adapter/NotificationPagingAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package com.teamdontbe.feature.notification.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.paging.PagingDataAdapter
+import com.teamdontbe.domain.entity.NotiEntity
+import com.teamdontbe.feature.databinding.ItemNotificationFeedBinding
+import com.teamdontbe.feature.notification.adapter.NotificationAdapter.Companion.NotificationAdapterDiffCallback
+
+class NotificationPagingAdapter(
+    private val click: (NotiEntity, Int) -> Unit = { _, _ -> },
```

Comment:
```suggestion
    private val click: (NotiEntity, Int) -> Unit
```
p3 : 클릭만 담당하는 거면.. 아마 이렇게 해도 될것 같아요!

Reply:
그러네요!